app/theme.ts

```diff
@@ -2,16 +2,22 @@ import {
 	extendTheme,
 	withDefaultColorScheme,
 	ThemeConfig,
+	theme,
 } from "@chakra-ui/react";
 
 const config: ThemeConfig = {
-	initialColorMode: "system",
+	initialColorMode: "light",
 };
 
 export default extendTheme(
 	{ config },
+	{
+		colors: {
+			brand: theme.colors.purple,
```

Comment:
You sure we want to start customizing the color scheme this early? IMO it's quite fine as-is.

Reply:
I changed it to purple for testing, should have changed it back to blue, my bad ;-;